### PR TITLE
Update the description of the portal storm "person"

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary
None

#### Purpose of change

The concept behind this encounter is very good, but the description wasnt the best (also had a grammar error). 
So I tried to improve it a bit

#### Describe the solution

New description

#### Describe alternatives you've considered

Just fix the grammar.

#### Testing

Words

### CODE REVIEW

**Comment by GuardianDll** [comment]
I dont really like it lost his "yep, thats 100% human, nothing wrong here" vibe, but if its better i can accept it

File: data/json/monsters/nether.json [hunk 1]
```diff
@@ -1291,8 +1291,8 @@
   {
     "id": "mon_hallucinator",
     "type": "MONSTER",
-    "name": { "str": "person" },
-    "description": "A common human, that looks exactly like human do.  You can see it.  It can see you.  You know it knows you can see it.  It knows you know you can see him.  It moves towards you.",
+    "name": { "str": "storm roamer" },
+    "description": "Their greater than human countenance festers in the deepest recesses of your mind.  It grows ever closer.",
     "//": "it must be spooky, but god im bad in this stuff",
```
Comment (Fris0uman) [inline data/json/monsters/nether.json:1296]:
```suggestion
```
No need for this comment once the description is redone

**Comment by Fris0uman** [comment]
The rewrite seems to change the monster from "this is the concept of a person" to "this is a spooky inhabitant of the nether"

**Comment by Nebnis** [comment]
Here is my spin off for it.

"An ordinary Human. It approaches you as certainly as the dreading feeling that they know exactly where you are."

**Comment by John-Candlebury** [comment]
I was thinking that the “more than human countenance “ would be a sufficient  implication that the apparition is a person. I can be more explicit  i suppose  although I like that my proposal leaves much unexplained .

**Comment by Nebnis** [comment]
Well, "More than human appearance" implies it is something otherworldly for me. The rest of the phrase make look like it is psychically attacking you.

**Comment by tornikeo** [comment]
Damn. I liked the original more. :/ To me, it's the weirdness is what makes it spooky.

**Comment by MNG-cataclysm** [comment]
I prefer the original description too. It has a more ominous effect which triggers the uncanny valley, while the new one doesn’t have the same flair.

**Comment by chrispikula** [comment]
I wouldn't mind the a slight name change, as it makes searching for it a right pain, but, yeah, agreed overall that the creature should be described as looking like a normal person.

**Comment by Sutremaine** [comment]
Break it even more. :)

 "A common human, that looks exactly like human do.  You can see it.  it sees you.  You know it knows you can see it.  It knows you know you can see him you see her you know It you.  The common human approaches."

**Comment by GuardianDll** [comment]
I see this one stuck in some middle position
Maybe just slightly polish its description?
`Its a normal human. It looks like a normal human would. You can see it. It can see you. It knows you can see it. Its coming closer.` from reddit looks good example for me

**Comment by GuardianDll** [comment]
May i ask you just fix a grammar? i believe it will lost big part of it's charm if description would be replaced

File: data/json/monsters/nether.json [hunk 2]
```diff
@@ -1292,8 +1292,7 @@
     "id": "mon_hallucinator",
     "type": "MONSTER",
     "name": { "str": "person" },
-    "description": "A common human, that looks exactly like human do.  You can see it.  It can see you.  You know it knows you can see it.  It knows you know you can see him.  It moves towards you.",
-    "//": "it must be spooky, but god im bad in this stuff",
+    "description": "Their dubiously human countenance festers in the deepest recesses of your mind.  They grow ever closer.",
```
Comment (nornagon) [inline data/json/monsters/nether.json:1295]:
```suggestion
    "description": "A common human. It looks exactly like a human looks. You can see them. They can see you. You know they know you can see them. They know you know you can see them. They move towards you.",
```

**Comment by John-Candlebury** [comment]
I do not consider the original description, nor the repetition about visuals to be charming or very good either. Feel free to open a pr with your suggestion.

**Pull request closed without merging**